providers/network/resources/tls.go

```diff
@@ -28,19 +28,21 @@ var rexUrlDomain = regexp.MustCompile(regex.UrlDomain)
 // If no port is set, we estimate what it might be from the scheme.
 // If that doesn't help, we set it to 443.
 func connTlsPort(conn *connection.HostConnection) int64 {
-	if conn.Conf.Port != 0 {
-		return int64(conn.Conf.Port)
+	port := conn.Conf.Options["port"]
+	portNumber, _ := strconv.ParseInt(port, 10, 64)
+	if portNumber != 0 {
+		return int64(portNumber)
 	}
 
-	if conn.Conf.Runtime == "" {
+	if conn.Conf.Options["runtime"] == "" {
```

Comment:
same here

Reply:
The first one I agree, this second one I don't see what you mean